elytron/src/main/java/org/wildfly/extension/elytron/CertificateAuthorityAccountDefinition.java

```diff
@@ -78,10 +78,11 @@
  */
 class CertificateAuthorityAccountDefinition extends SimpleResourceDefinition {
     private static final String DIRECTORY = "directory";
+    private static final String URL = "http://localhost:4001/";
 
     enum CertificateAuthority {
-
-        LETS_ENCRYPT("LetsEncrypt", "https://acme-v02.api.letsencrypt.org/" + DIRECTORY, "https://acme-staging-v02.api.letsencrypt.org/" + DIRECTORY);
+        //TODO: replace with the configuration
```

Comment:
I will remove changes in this file once the https://issues.jboss.org/browse/EAP7-1150 is fixed.